### PR TITLE
Apply clippy lint.

### PR DESCRIPTION
**Pull request by ffranr**

### CODE REVIEW

File: crev-data/src/proof/mod.rs [hunk 1]
```diff
@@ -259,13 +259,11 @@ impl Proof {
...
+                        } else if is_signature_line(line) {
+                            self.stage = Stage::Signature;
                         } else {
-                            if is_signature_line(line) {
-                                self.stage = Stage::Signature;
-                            } else {
-                                self.body += line;
-                                self.body += "\n";
-                            }
+                            self.body += line;
```
Comment (dpc) [inline crev-data/src/proof/mod.rs:265]: I actually disagree with this lint in clippy. Oftentimes it makes it less-clear what's going on, when there are multiple nested logical choices. By flattening them, some information is lost. But not a blocker.

File: crev-data/src/proof/mod.rs [hunk 2]
```diff
@@ -259,13 +259,11 @@ impl Proof {
                                 self.body += line;
                                 self.body += "\n";
                             }
+                        } else if is_signature_line(line) {
```
Comment (dpc) [inline crev-data/src/proof/mod.rs:262]: I actually disagree with this lint in clippy. Oftentimes it makes it less-clear what's going on, when there are multiple nested logical choices. Eample:

if A {

} else { 
   if B {
   } else {
   }
}

B is a sub-case of "not A" and by flattening them, some information is lost, IMO. But not a blocker.

Comment (dpc) [reply crev-data/src/proof/mod.rs:262]: Ideally, we could disable that lint globally in the project.

Comment (ffranr) [reply crev-data/src/proof/mod.rs:262]: I see what you mean, I'll disable it globally.

Comment (ffranr) [reply crev-data/src/proof/mod.rs:262]: There isn't currently a way of disabling this particular lint check globally, see: https://github.com/EmbarkStudios/rust-ecosystem/issues/22

I will instead skip this lint for this PR.

Comment (dpc) [reply crev-data/src/proof/mod.rs:262]: We could just add it in every affected crate.

**Pull request merged**